Bottom: 1d6583921c5643f83bfba565b9afa8352556cd08
Top:    448671357930c84fc85ba381d32717633a48a2b0
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-11 13:46:46 +0800

Refresh of autofs-5.0.7-probe-each-nfs-version-in-turn-for-singleton-mounts.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 39d7889..48e9806 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -48,6 +48,7 @@
 - fix master map mount options matching.
 - fix master map bogus keywork match.
 - fix fix map entry duplicate offset detection.
+- probe each nfs version in turn for singleton mounts.
 
 25/07/2012 autofs-5.0.7
 =======================
